### PR TITLE
Html forms

### PR DESCRIPTION
**Pull request by OSAdams**
image
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+    <label for="code-question">Why is coding cool?</label>
+    <div>
+      <select name="code-question" id="code-question">
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <p>
```
Comment (Zhiwutian) [inline html-forms/index.html:28]: this should be a div not a p tag

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <label for="favorite-language">Favorite Language &lpar;choose one&rpar;</label>
+    </p>
+    <input type="radio" id="html" name="favorite-language" value="html" checked="checked"> HTML
+    <input type="radio" id="css" name="favorite-language" value="css"> CSS
+    <input type="radio" id="javascript" name="favorite-language" value="javascript"> JavaScript
+    <p>
```
Comment (Zhiwutian) [inline html-forms/index.html:34]: same as above

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      <select name="code-question" id="code-question">
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <p>
+      <label for="favorite-language">Favorite Language &lpar;choose one&rpar;</label>
+    </p>
+    <input type="radio" id="html" name="favorite-language" value="html" checked="checked"> HTML
```
Comment (Zhiwutian) [inline html-forms/index.html:31]: you need a label for each of these with match for and id attributes, or you can wrap the inputs with the label

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

**Comment by OSAdams** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**